### PR TITLE
Add shorter type synonyms for all eras

### PR DESCRIPTION
**Pull request by lehins**
**NOTE** - This PR also include changes from #3016, so it might be best to wait for that one to be reviewed and merged before reviewing this PR

Here are the type synonyms:
```haskell
type Shelley = ShelleyEra StandardCrypto

type Allegra = AllegraEra StandardCrypto

type Mary = MaryEra StandardCrypto

type Alonzo = AlonzoEra StandardCrypto

type Babbage = BabbageEra StandardCrypto

type Conway = ConwayEra StandardCrypto
```

### CODE REVIEW

File: eras/shelley-ma/test-suite/test/Test/Cardano/Ledger/Mary/Translation.hs [hunk 1]
```diff
@@ -12,17 +12,14 @@ where
 import Cardano.Binary
   ( ToCBOR (..),
   )
+-- import Allegra EraGen instance
```
Comment (JaredCorduan) [inline eras/shelley-ma/test-suite/test/Test/Cardano/Ledger/Mary/Translation.hs]: is this that helpful? :thinking:

File: eras/shelley/test-suite/test/Test/Cardano/Ledger/Shelley/Fees.hs [hunk 2]
```diff
@@ -89,13 +85,8 @@ import Test.Cardano.Ledger.Shelley.Utils
...
 
-sizeTest ::
-  (HasCallStack, Cr.Crypto c) =>
-  proxy c ->
-  BSL.ByteString ->
-  ShelleyTx (ShelleyEra c) ->
-  Assertion
-sizeTest _ b16 tx = do
+sizeTest :: HasCallStack => BSL.ByteString -> ShelleyTx Shelley -> Assertion
+sizeTest b16 tx = do
```
Comment (JaredCorduan) [inline eras/shelley/test-suite/test/Test/Cardano/Ledger/Shelley/Fees.hs:92]: nice!

**Review by JaredCorduan — APPROVED** [review]
fantastic clean up! I just reviewed the single commit that is not in 3016.

File: eras/shelley-ma/test-suite/test/Test/Cardano/Ledger/Mary/Translation.hs (hunk 1 above)
Comment (lehins) [reply eras/shelley-ma/test-suite/test/Test/Cardano/Ledger/Mary/Translation.hs]: Yeah that is useless :smile: I think it was accidental copy and paste at some point. I didn't even notice it. Looks like ormolu moved it together with the changed import. I'll get rid of it

**Pull request merged**